.helm/ecamp3/values.yaml

```diff
@@ -91,10 +91,25 @@ browserless:
   image:
     repository: "docker.io/browserless/chrome"
     pullPolicy: IfNotPresent
-    tag: "1.53-puppeteer-13.6.0"
+    tag: "1.56.0-puppeteer-18.0.5"
   service:
     type: ClusterIP
     port: 3000
+  maxConcurrentSessions: 1
```

Comment:
Shouldn't this be higher than 1 on deployments? A low default is okay for test deployments, but especially on alpha and prod this will have to be set higher, no?

Reply:
Yes, on prod we definitely want to have multiple parallel sessions. How much exactly we probably have to figure out what the peak load will really be.

For alpha, I've set it to 3 parallel sessions for the moment: https://github.com/ecamp/ecamp3/pull/3119/commits/4ed3557e1869d364e576ea2eec6b724a7ee27c66